docs/user-guide/kubectl-cheatsheet.md

```diff
@@ -5,22 +5,54 @@ assignees:
 - krousey
 
 ---
-An assortment of compact kubectl examples
+A cheat sheet for `kubectl`.
```

Comment:
Why? The title is already "Kubectl cheat sheet", which is not self-explanatory for non-English speakers.

Suggest changing to "This document contains a quick reference for common tasks you can accomplish with `kubectl`."